spec/support/matchers.rb

```diff
@@ -42,6 +42,7 @@
       next if a[:lang]
       page = a[:href]
       link_path = URI::parse(page).path
+      skip if link_path.match(/partners/)
```

Comment:
I'm not familiar with `skip`. Is it the same as `next` ?

Reply:
I'm actually surprised this worked. I was thinking of RSpec [skip](https://relishapp.com/rspec/rspec-core/docs/pending-and-skipped-examples/skip-examples#skipping-using-%60skip%60-inside-an-example). I'm going to change it to next.